### PR TITLE
New dribbler

### PR DESCRIPTION
**Pull request by DarioC02**

### CODE REVIEW

File: Core/Inc/dribbler_board/Util/AC_DC_util.h
Comment (JWillegers) [inline Core/Inc/dribbler_board/Util/AC_DC_util.h:1]: Change CUR_DRIBBLER to CURRENT_DRIBBLER and VOLT_DRIBBLER to VOLTAGE_DRIBBLER

File: Core/Inc/dribbler_board/peripherals/dribbler.h [hunk 1]
```diff
@@ -13,22 +13,38 @@
...
 #include <math.h>
 #include "control_util.h"
-#include "logging.h"
 
 ///////////////////////////////////////////////////// PUBLIC FUNCTION DECLARATIONS
 
-uint32_t set_speed;
+extern uint32_t encoder_value;
+
+#define CURRENT_THRESHOLD 600
```
Comment (JWillegers) [inline Core/Inc/dribbler_board/peripherals/dribbler.h]: please add documentation either here or on the wiki why this is the treshold

File: Core/Src/dribbler_board/Util/AC_DC_util.c
Comment (JWillegers) [inline Core/Src/dribbler_board/Util/AC_DC_util.c:1]: remove file, is unneccesary

File: Core/Src/dribbler_board/dribbler_board.c
Comment (JWillegers) [inline Core/Src/dribbler_board/dribbler_board.c:1]: please clean up file, remove empty functions and delete old, commented out code

File: Core/Src/dribbler_board/peripherals/ballSensor.c
Comment (JWillegers) [inline Core/Src/dribbler_board/peripherals/ballSensor.c:1]: could you add a test in ballsensor_init() to see if ball sensor is working?

**Review by JWillegers — CHANGES_REQUESTED** [review]
see comments

**Review by jggithub17 — APPROVED** [review]
Reviewed the last three commits, the commits before that were approved by Jonathan

**Review by JWillegers — APPROVED** [review]

**Pull request merged**